public/app/core/components/AppChrome/NavToolbar.tsx

```diff
@@ -29,8 +31,9 @@ export function NavToolbar({
   onToggleSearchBar,
   onToggleKioskMode,
 }: Props) {
+  const homeNav = useSelector((state) => state.navIndex)[HOME_NAV_ID];
   const styles = useStyles2(getStyles);
-  const breadcrumbs = buildBreadcrumbs(sectionNav, pageNav);
+  const breadcrumbs = buildBreadcrumbs(homeNav, sectionNav, pageNav);
```

Comment:
can we get home nav node from sectionNav.main.parentItem ?

Reply:
@torkelo yes, but only if `sectionNav.main` is always an object directly below home. i guess that's what it _should_ be, but right now sometimes it's not. 

will leave this for a further refactor.